trypurescript.cabal

```diff
@@ -15,21 +15,21 @@ data-dir: ""
 executable trypurescript
     build-depends: base ==4.*,
                    aeson -any,
-                   bytestring >=0.10.0.2 && <0.11,
+                   bytestring >=0.10.12.0 && <0.11,
                    data-default -any,
                    directory -any,
                    filepath -any,
                    Glob -any,
-                   scotty -any,
+                   scotty ==0.12,
                    purescript,
                    purescript-cst,
                    containers -any,
                    http-types >= 0.8.5,
```

Comment:
@hdgarrood Before I merge this, should `http-types` be updated to `-any` or to an the [`lts-17.6` package set](https://www.stackage.org/lts-17.6)'s version, which is `0.12.3`?

And is it correct that `-any` will use whichever dependencies work with `purescript` and the other versions dependencies? I'm not familiar with this syntax unfortunately.

Reply:
Thinking about this a little more I think we might as well use `-any` for all of the dependencies here. This project is only ever going to be built by `stack`, and the stack.yaml file already pins everything down to a specific version, so additionally specifying versions here doesn’t really add anything.

It’s more important to provide accurate version bounds when you’re making a library which other people are depending on - this part of the cabal file does the same job as the dependencies part of a bower.json or package.json file in a JS library. But that’s not what we are doing here so it doesn’t really matter.